WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/NewBlogDetailHeaderView.swift

```diff
@@ -0,0 +1,175 @@
+class NewBlogDetailHeaderView: UIView, BlogDetailHeader {
+
+    @objc weak var delegate: BlogDetailHeaderViewDelegate?
+
+    // Temporary method for migrating to NewBlogDetailHeaderView
+    @objc
+    var asView: UIView {
+        return self
+    }
+
+    private let titleButton: SpotlightableButton = {
+        let button = SpotlightableButton(type: .custom)
+        button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.title2, fontWeight: .bold)
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+        button.titleLabel?.lineBreakMode = .byTruncatingTail
+        button.setTitleColor(.text, for: .normal)
+        button.addTarget(self, action: #selector(titleButtonTapped), for: .touchUpInside)
+        return button
+    }()
+
+    private let subtitleLabel: UILabel = {
+        let label = UILabel()
+        label.font = WPStyleGuide.fontForTextStyle(.subheadline)
+        label.textColor = UIColor.textSubtle
+        label.adjustsFontForContentSizeCategory = true
+        return label
+    }()
+
+    private let siteIconView: SiteIconView = {
+        let view = SiteIconView(frame: .zero)
+        return view
+    }()
+
+    @objc var updatingIcon: Bool = false {
```

Comment:
One small observation on this, since we are refactoring: it's perfectly fine, but if the purpose of this state variable is only to start and stop the spinner, can we have a `startSpinner` (or activity indicator) and `stopSpinner` called directly instead of setting the variable? This might be more readable when the code gets more complex in the future

Reply:
Yeah, but would you mind if I tackled this in my next `NewBlogDetailHeaderView` PR?  It would allow me to keep this particular PR the simplest if possible.

Reply:
Sure, makes total sense!!